packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -2183,4 +2191,26 @@ describe('ReactDOMInput', () => {
     ReactDOM.render(<input type="text" defaultValue={null} />, container);
     expect(node.defaultValue).toBe('');
   });
+
+  it('should notice input changes when reverting back to original value', () => {
+    const log = [];
+    function onChange(e) {
+      log.push(e.target.value);
+    }
+    ReactDOM.render(
+      <input type="text" value="" onChange={onChange} />,
+      container,
+    );
+    ReactDOM.render(
+      <input type="text" value="a" onChange={onChange} />,
+      container,
+    );
+
+    const node = container.firstChild;
+    setUntrackedValue.call(node, '');
+    dispatchEventOnNode(node, 'input');
+
+    expect(log.splice(0, log.length)).toEqual(['']);
```

Comment:
why the splice?

Reply:
ah I thought I might have multiple asserts on the log and wanted to clear it